anchorecli/cli/utils.py

```diff
@@ -6,14 +6,13 @@
 import yaml
 import logging
 import dateutil.parser
-import struct
 import textwrap
 import base64
 
 try:
-    from urllib.parse import quote_plus,unquote_plus
-except:
-    from urllib import quote_plus,unquote_plus
+    from urllib.parse import quote_plus, unquote_plus
+except ImportError:
```

Comment:
if we find other imports like this it might make sense to add a `compat.py` that checks for python version instead. I've seen issues even when guarding explicitly against `ImportError` that are hard to debug